mmaction/datasets/pipelines/augmentations.py

```diff
@@ -42,6 +50,166 @@ def _init_lazy_if_proper(results, lazy):
+            dict(
+                type='SomeOf',
+                n=(0, 3),
+                children=[
+                    dict(
+                        type='OneOf',
+                        children=[
+                            dict(type='GaussianBlur', sigma=(0, 0.5)),
+                            dict(type='AverageBlur', k=(2, 7)),
+                            dict(type='MedianBlur', k=(3, 11))
+                        ]),
+                    dict(
+                        type='OneOf',
+                        children=[
+                            dict(
+                                type='Dropout', p=(0.01, 0.1),
+                                per_channel=0.5),
+                            dict(
+                                type='CoarseDropout',
+                                p=(0.03, 0.15),
+                                size_percent=(0.02, 0.05),
+                                per_channel=0.2),
+                        ]),
+                    dict(
+                        type='AdditiveGaussianNoise',
+                        loc=0,
+                        scale=(0.0, 0.05 * 255),
+                        per_channel=0.5),
+                ]),
+        ]
+
+    def imgaug_builder(self, cfg):
+        """Import a module from imgaug.
+
+        It inherits some of :func:`build_from_cfg` logic.
+
+        Args:
+            cfg (dict): Config dict. It should at least contain the key "type".
+
+        Returns:
+            obj: The constructed object.
+        """
+        import inspect
+
+        assert isinstance(cfg, dict) and 'type' in cfg
+        args = cfg.copy()
+
+        obj_type = args.pop('type')
+        if mmcv.is_str(obj_type):
+            obj_cls = getattr(iaa, obj_type)
+        elif inspect.isclass(obj_type):
+            assert obj_type == getattr(iaa, obj_type.__name__)
+            obj_cls = obj_type
+        else:
+            raise TypeError(
+                f'type must be a str or valid type, but got {type(obj_type)}')
+
+        if 'children' in args:
+            args['children'] = [
+                self.imgaug_builder(child) for child in args['children']
+            ]
+
+        return obj_cls(**args)
+
+    def __repr__(self):
+        repr_str = self.__class__.__name__ + f'(transforms={self.transforms})'
+        return repr_str
+
+    def __call__(self, results):
+        cur_aug = self.aug.to_deterministic()
+
+        results['imgs'] = [
+            cur_aug.augment_image(frame) for frame in results['imgs']
+        ]
+        img_h, img_w, _ = results['imgs'][0].shape
+
+        if 'gt_bboxes' in results:
+            bbox_list = [
+                BoundingBox(x1=bbox[0], y1=bbox[1], x2=bbox[2], y2=bbox[3])
+                for bbox in results['gt_bboxes']
+            ]
+            bboxes = BoundingBoxesOnImage(
+                bbox_list, shape=results['img_shape'])
+            bbox_aug = cur_aug.augment_bounding_boxes([bboxes])[0]
```

Comment:
What are the other indexes other than 0? Maybe use
```
bbox_aug, *_ = cur_aug.augment_bounding_boxes([bboxes])
```
for readability as others are not familiar with iaa.

Reply:
`aug.augment_bounding_boxes` and `aug.augment_images` are designed for augmenting a batch of images. Each image has its corresponding bboxes and each image has different augmenter args. 

A typical use case is shown as follows. A/Abb, B/Bbb, C/Cbb have different rotate degree.

```python
Abb = Bbb = Cbb = BoundingBoxesOnImage([
    BoundingBox(1, 1, 9, 9)], (10, 10))
seq = iaa.Rotate()
seq_det = seq.to_deterministic()
imgs_aug = seq_det.augment_images([A, B, C])
bbs_aug = seq_det.augment_bounding_boxes([Abb, Bbb, Cbb])
```

We use `augment_image` instead of `augment_images` to use same aug for all images.